src/main/scala/sbtdatabricks/DatabricksPlugin.scala

```diff
@@ -137,6 +145,73 @@ object DatabricksPlugin extends AutoPlugin {
     else set
   }
 
+  
+  @annotation.tailrec
+  private def onClusterStatus(client: DatabricksHttp,
+                              clusterId: String, 
+                              desiredStatus: DBCClusterStatus,
+                              pollTime: Long): Option[ClusterId] = {
+    val cluster = client.checkCluster(clusterId).get
```

Comment:
if cluster was `None` this will throw an exception. How about something like:
```scala
client.checkCluster(clusterId) match {
  case Some(cluster) if cluster.status == DBCClusterError.status =>
     sys.error(...)
  case Some(cluster) if cluster.status == desiredStatus.status =>
     Some(ClusterId(cluster.id))
  case _ =>
     Thread.sleep(pollTime)
     onClusterStatus(client, clusterId, desiredStatus, pollTime)
}
```